packages/forms/src/UIForm/fields/NestedListView/NestedListView.utils.js

```diff
@@ -0,0 +1,71 @@
+/**
+ * Build the ListView component props to match current state's values
+ * @param {Array} items
+ * @param {Object} value
+ * @param {String} searchCriteria
+ * @returns {Array}
+ */
+export function getDisplayedItems(items, value, searchCriteria) {
+	let textFilter;
+	if (searchCriteria) {
+		textFilter = ({ label }) => label.toLowerCase().includes(searchCriteria.toLowerCase());
+	}
+
+	return (
+		items
+			.map(item => {
+				// Add boolean "checked" to checked children
+				const checkedChildren = item.children.map(child => ({
```

Comment:
This var name isn't what's really inside. There could be unchecked children

Reply:
Hence the comment. Plus, 100 char per line limit is easily reached